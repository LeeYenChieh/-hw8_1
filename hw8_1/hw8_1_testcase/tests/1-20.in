pG9nO GdhnIrbT5i8oDzPpnwWSKLgAyZDhMAm8TBHfY1kcadW
Hy9Fg1WSMnCznqNwTyGuKLklAiGoR3ViXoo4WuIua WuXFmaCT
Ayo oOvJNdm26zKuk3eoNYw2wNvH1khpgSdSXD7tuMVqEuSeg
wEimPAhi4n9J9lQvcsT275mqSS zUCzH9YypM43eviK
EzjvYG5daF577Ow9NhNKg6V3hx0x2TyrifVAcGZOS5WXsAgzQCdkrkmLSnNoNu 3P0PvQxkcAL48nkEyBzFhsja03Qou308KXRmmphsGzmVBGqn9p
HRwKxDuFdwDtGZRmnC0OtonIuPYdgiIFoiw3sETXa4Lt1Rc4S5hMCTG6eeGY3A Oa8YgUjThtJj
Xk7E9jtxWCIxwH11xK8XK pjLbIxIjCGblacRVLeq3ybP7YD5tkcqDWj7nvmFukDi0
Ayo aoMWiHZLED6a1MpgKljFTPpYHW357XlmM6y0
pG9nO display
wEimPAhi4n9J9lQvcsT275mqSS display
wEimPAhi4n9J9lQvcsT275mqSS 4Ks8hUhUNLCgHFqyKDGFrw22Rwhxl4Uy0NlPWM
lXyx4XtSXChI9dyjOwrFQcrOZI9FXQ0T6q33O JKg5IjS1mqZDlgYDneZX8pMj7N7XIxWQrNJ4fQiW5LTq2ZNyy4u
lXyx4XtSXChI9dyjOwrFQcrOZI9FXQ0T6q33O Xsl0wNzHu0piAQrUbBQOuz4cEkgu56470r7fvrgCKdEdKPeo4oE7
RVYsbgY9DmIDDZWxTKSiOlnA1sS9uzkdoEY9GsoJa 0tWAlHoGwl8T6iAPivw5qlwv4HYtKmZPN95sGNr8VjhJcn1wVUfJr
ZRbruHcXMnVBURVdlGxiGHNi Jrhy104iIfCqPlDHs25adsEk
