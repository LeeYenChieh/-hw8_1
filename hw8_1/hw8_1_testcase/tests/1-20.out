GdhnIrbT5i8oDzPpnwWSKLgAyZDhMAm8TBHfY1kcadW 
zUCzH9YypM43eviK 
